pkg/sidecarlogresults/sidecarlogresults.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sidecarlogresults
```

Comment:
I was wondering where it would fit. I placed it here originally because I thought it belonged in a package parallel to `termination` which in turn was directly under pkg. 
```
$ ls
apis               container          entrypoint         list               pipelinerunmetrics reconciler         sidecarlogresults  substitution       trustedresources
artifacts          controller         git                matrix             pod                remote             spire              taskrunmetrics     workspace
client             credentials        internal           names              pullrequest        resolution         status             termination
```

If you can think of an obvious place, do let me know.

Reply:
moved to `pkg/internal/sidecarlogresults`. Thanks!!!